sample_data_config/gmail-data/data-schema/gmail_label_list_schema.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "mode": "REQUIRED",
+        "name": "labelId",
+        "type": "STRING"
+    },
+    {
+        "mode": "REQUIRED",
+        "name": "labelName",
+        "type": "STRING"
+    },
+    {
+        "mode": "NULLABLE",
+        "name": "messageListVisibility",
+        "type": "STRING"
+    },
+    {
+        "mode": "NULLABLE",
+        "name": "labelListVisibility",
+        "type": "STRING"
+    },
+    {
+        "mode": "NULLABLE",
+        "name": "labelType",
+        "type": "STRING"
+    },
+    {
+        "mode": "NULLABLE",
+        "name": "labelColour",
```

Comment:
Are the field ids in here returned by the API? I only noticed it because names are usually americanised. The API docs have slightly different names, e.g. `color`: https://developers.google.com/gmail/api/reference/rest/v1/users.labels#Label (is that the correct API docs?)
(But you used the API and will know better)

Reply:
I think I am still confused how the import with CSV works. We don't seem to be providing the schema to the bigquery table. It seems odd if the bigquery import itself was to assume that the columns in the CSV match exactly the column order of the BigQuery table (but everything is possible). If that was really the case, then we could probably just select the columns in the right order you don't want to switch to JSONL.

Another thing I haven't thought of before is that the default mode is to append. i.e. every time you run the pipeline, it should append updated labels. If we keep that append mode, then we need at least an `imported_timestamp`.